### PR TITLE
Add tooling for creating a stub universe

### PR DESCRIPTION
**Pull request by benclarkwood**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    echo "Using DOCKER_IMAGE : $DOCKER_IMAGE"
+else
+	echo "Building docker image..."
+	# docker publish to user docker repo (e.g. DOCKER_REPO envvar)
+	: ${DOCKER_REPO?"Need to set DOCKER_REPO env var as DOCKER_IMAGE is not specified"}
+
+	DOCKER_IMAGE="${DOCKER_REPO}/jenkins-dev:latest"
+	# docker build and publish
+	docker build -t "${DOCKER_IMAGE}" .
+	docker push "${DOCKER_IMAGE}"
```
Comment (colin-msphere) [inline build.sh]: Do we want this able to overwrite the CI published image?

File: build.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+if [ -n "${DOCKER_IMAGE}" ]; then
+	# DOCKER_IMAGE envvar is user supplied
+    echo "Using DOCKER_IMAGE : $DOCKER_IMAGE"
+else
+	echo "Building docker image..."
+	# docker publish to user docker repo (e.g. DOCKER_REPO envvar)
+	: ${DOCKER_REPO?"Need to set DOCKER_REPO env var as DOCKER_IMAGE is not specified"}
+
+	DOCKER_IMAGE="${DOCKER_REPO}/jenkins-dev:latest"
```
Comment (colin-msphere) [inline build.sh]: `latest` or the git sha?

Comment (benclarkwood) [reply build.sh]: git sha would be best

fixing.

**Comment by benclarkwood** [comment]
Had some wrong settings in the `docker` section after UCR switch. Dropped those.

I didn't take out any of the config yet.

**Pull request merged**